LDAR_Sim/src/sensors/default_sensor.py

```diff
@@ -18,24 +18,32 @@
 ------------------------------------------------------------------------------
 """
 
+from typing import Union
 import numpy as np
 from virtual_world.sites import Site
 
 
 class DefaultSensor:
-    def __init__(self, mdl: float, quantification_error: float) -> None:
-        self._mdl: float = mdl
+    def __init__(self, mdl: Union[list[float], float], quantification_error: float) -> None:
+        if isinstance(mdl, (float, int)):
+            self._mdl: float = mdl
+        elif isinstance(mdl, list):
+            self._mdl: float = mdl[0]
+            if len(mdl) == 4:
```

Comment:
Why are we checking that the default sensor MDL length is equal to 4?
The default sensor is a threshold. It makes no sense to do this check for the default sensor. 
If anything if the default sensor is given a list of floats with length greater than 1, printing out an error might be better.